src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -168,6 +168,7 @@ bool OutputStateMachineEngine::ActionPassThroughString(const std::wstring_view s
     if (_pTtyConnection != nullptr)
     {
         const auto hr = _pTtyConnection->WriteTerminalW(string);
+
         LOG_IF_FAILED(hr);

```

Comment:
Again not important, but I'm curious whether there was a reason for this, or just the cat getting onto the keyboard?

Reply:
Honestly, couldn't tell you :p